api/base_controller.go

```diff
@@ -195,6 +203,30 @@ func (c *BaseController) GetSingleObject(r *web.Request) (*web.Response, error)
 	return util.NewJSONResponse(http.StatusOK, object)
 }
 
+// GetOperation handles the fetching of a single operation with the id specified for the specified resource
+func (c *BaseController) GetOperation(r *web.Request) (*web.Response, error) {
+	objectID := r.PathParams[PathParamResourceID]
+	operationID := r.PathParams[PathParamID]
+
+	ctx := r.Context()
+	log.C(ctx).Debugf("Getting operation with id %s for object of type %s with id %s", operationID, c.objectType, objectID)
+
+	byOperationID := query.ByField(query.EqualsOperator, "id", operationID)
+	byObjectID := query.ByField(query.EqualsOperator, "resource_id", objectID)
+	var err error
+	ctx, err = query.AddCriteria(ctx, byObjectID, byOperationID)
+	if err != nil {
+		return nil, err
+	}
+	criteria := query.CriteriaForContext(ctx)
+	operation, err := c.repository.Get(ctx, types.OperationType, criteria...)
```

Comment:
You have to strip the tenant identifier label if it's in the current criteria, otherwise the DB operation will fail with non-existant entity.

Reply:
Well I actually rely on it to be there so that calls with tenant access cannot see global operations. See how operations for tenant scoped resources are created in the authz tests for reference

Reply:
Yeah, sure, but it still breaks the DB Get, so it has to be stripped away before making the DB operation

Reply:
Its needed in order to disallow tenant access to see operations for global resources